src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -162,7 +162,8 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, reimbursementAcc
 
     const hasMembersError = PolicyUtils.hasEmployeeListError(policy);
     const hasPolicyCategoryError = PolicyUtils.hasPolicyCategoriesError(policyCategories);
-    const hasGeneralSettingsError = !isEmptyObject(policy?.errorFields?.generalSettings ?? {}) || !isEmptyObject(policy?.errorFields?.avatarURL ?? {});
+    const hasGeneralSettingsError =
```

Comment:
Should we add `policy?.errorFields?.address`? Maybe the FE doesn't throw this field but the BE can return it. Let's add it to keep consistency with other fields